requirements.txt

```
@@ -0,0 +1,6 @@
+numpy
+pytest
+tqdm
+pre-commit
```

Pretty sure these are still unnecessary

```suggestion
tqdm
```